--- origsrc/dasher-4.7.3/Src/Common/AppSettingsHeader.h	2008-04-07 12:25:52.000000000 -0500
+++ src/dasher-4.7.3/Src/Common/AppSettingsHeader.h	2008-05-30 01:16:32.453125000 -0500
@@ -3,7 +3,7 @@
 
 #include "../DasherCore/Parameters.h"
 #ifndef _WIN32
-#include "../../config.h"
+#include "config.h"
 #endif
 
 enum { 
--- origsrc/dasher-4.7.3/Src/DasherCore/Parameters.h	2008-04-07 12:25:31.000000000 -0500
+++ src/dasher-4.7.3/Src/DasherCore/Parameters.h	2008-05-30 01:16:32.484375000 -0500
@@ -24,7 +24,7 @@
 #include <string>
 
 #ifndef _WIN32
-#include "../../config.h"
+#include "config.h"
 #endif
 
 // All parameters go into the enums here
